docs/src/donu.md

```diff
@@ -96,24 +96,29 @@ Example:
 }
 ```
 
-### `simulate-gsl` - Simulate a model using ODEs via the Gnu Scientific Library
+### `simulate` - simulate a model
 
 **Request:**
 
-| Field            | Type                     | Description                                                           |
-|------------------|--------------------------|-----------------------------------------------------------------------|
-| command          | string                   | Command - for this operation it will be the string `"simulate-gsl"`   |
-| definition       | model-def                | Definition of the model                                               |
-| start            | number                   | Start time of the simulation                                          |
-| end              | number                   | End time of the simulation                                            |
-| step             | number                   | Simulation time step size                                             |
-| parameters       | dict                     | Parameter values/initial conditions for the simulation                |
+| Field            | Type                     | Description                                                                           |
+|------------------|--------------------------|---------------------------------------------------------------------------------------|
+| command          | string                   | Command - for this operation it will be the string `"simulate"`                       |
+| sim-type         | string                   | (optional) Simulation engine - one of `"aj"`, `"discrete"`, `"gsl"` (default `"gsl"`) |
+| definition       | model-def                | Definition of the model                                                               |
+| start            | number                   | Start time of the simulation                                                          |
+| end              | number                   | End time of the simulation                                                            |
+| step             | number                   | Simulation time step size                                                             |
+| seed             | integer                  | (optional) Use this seed for random number generation/event selection                 |
```

Comment:
This is only used for discrete event simulation, correct? If so, it might be worth noting this in the documentation here (as well as any other fields that are specific to certain types of simulation).